front/src/applications/operationalStudies/components/Study/AddOrEditStudyModal.tsx

```diff
@@ -189,41 +189,63 @@ export default function AddOrEditStudyModal({ editionMode, study }: Props) {
               <div className="col-xl-6">
                 <div className="study-edition-modal-type mb-2">
                   <SelectImprovedSNCF
-                    title={
+                    label={
                       <div className="d-flex align-items-center">
                         <span className="mr-2">
                           <RiQuestionLine />
                         </span>
                         {t('studyType')}
                       </div>
                     }
-                    selectedValue={{
-                      key: currentStudy?.study_type,
-                      value: t(`studyCategories.${currentStudy?.study_type || 'nothingSelected'}`),
-                    }}
+                    value={
+                      currentStudy
+                        ? {
+                            id: currentStudy.study_type,
+                            label: t(
+                              `studyCategories.${currentStudy.study_type || 'nothingSelected'}`
+                            ).toString(),
+                          }
+                        : undefined
+                    }
```

Comment:
Why did you change that ? currentStudy is never null or undefined
```suggestion
                    value={{
                      id: currentStudy.study_type,
                      label: t(
                        `studyCategories.${currentStudy.study_type || 'nothingSelected'}`
                      ).toString(),
                    }}
```